README.md

```diff
@@ -33,6 +33,25 @@ Developers and anyone else with a PyTorch or TensorFlow neural network model at
 ## Attacks
 
 ### Property Inference Attack
+![plot](docs/Property_Interference_Attacks.png)
+
+The Property Inference Attack aims to detect patterns in the parameters of the target model including properties 
+which the model producer has not intended to reveal. Therefore, the adversary requires a 
+meta-classifier which predicts one of these properties he is interested in. 
```

Comment:
I would prefer gender-neutral language. You could use plural (attackers/adversaries) and then "their"

Reply:
This was addressed in commit 4faf96c.